frameworks/cassandra/tests/test_auth.py

```diff
@@ -43,16 +58,63 @@ def configure_package(configure_security: None) -> Iterator[None]:
         yield  # let the test session execute
     finally:
         sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
-        delete_secret(secret=config.PACKAGE_NAME + '/' + config.SECRET_VALUE)
+        delete_secret(secret=config.PACKAGE_NAME + "/" + config.SECRET_VALUE)
         # remove job definitions from metronome
         for job in test_jobs:
             sdk_jobs.remove_job(job)
 
 
 @sdk_utils.dcos_ee_only
 @pytest.mark.sanity
-def test_auth() -> None:
-    config.verify_client_can_write_read_and_delete_with_auth(
+def test_backup_and_restore_to_s3_with_auth() -> None:
+    key_id = os.getenv("AWS_ACCESS_KEY_ID")
+    if not key_id:
+        assert (
+            False
+        ), 'AWS credentials are required for this test. Disable test with e.g. TEST_TYPES="sanity and not aws"'
+    plan_parameters = {
+        "AWS_ACCESS_KEY_ID": key_id,
+        "AWS_SECRET_ACCESS_KEY": os.getenv("AWS_SECRET_ACCESS_KEY"),
+        "AWS_REGION": os.getenv("AWS_REGION", "us-west-2"),
+        "S3_BUCKET_NAME": os.getenv("AWS_BUCKET_NAME", "infinity-framework-test"),
+        "SNAPSHOT_NAME": str(uuid.uuid1()),
```

Comment:
Please 
- change `S3_BUCKET_NAME` to `infinity-artifacts-ci` and 
- prefix the `SNAPSHOT_NAME` with `autodelete7d/cassandra/` 

This is so that all the snapshots get loaded to the right CI bucket and we have a policy of deleting objects after 7days so we don't blow up storage.